test/log.test.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const tap = require('tap')
+const sinon = require('sinon')
+
+const coreStubs = {
+  debug: sinon.stub(),
+  error: sinon.stub(),
+  info: sinon.stub(),
+  warning: sinon.stub(),
+}
+const log = tap.mock('../src/log', {
+  '@actions/core': coreStubs
+})
+
+tap.afterEach(() => {
+  Object.values(coreStubs).map(stub => stub.resetHistory())
```

Comment:
conceptually more correct using forEach since you don't use the return value. I have no used tap's native module mocks, I wonder if there's a way to make this simpler instead of having to manually reset the mocks after each test

```suggestion
  Object.values(coreStubs).forEach(stub => stub.resetHistory())
```

Reply:
Yep, I guess we can just call `sinon.restore()` from what I'm seeing on the docs. I'll try to improve that.